src/include/mpi.h.in

```diff
@@ -856,7 +856,7 @@ typedef int (MPIX_Grequest_wait_function)(int, void **, double, MPI_Status *);
                                            be initialized */
 #define MPI_T_ERR_INVALID_INDEX     62  /* The index is invalid or
                                            has been deleted  */
-#define MPI_T_ERR_INVALID_ITEM      63  /* Item index queried is out of range */
+#define MPI_T_ERR_INVALID_ITEM      MPI_T_ERR_INVALID_HANDLE  /* Deprecated */
```

Comment:
Is it `MPI_T_ERR_INVALID_HANDLE ` or `MPI_T_ERR_INVALID_INDEX`? 

It is a bit ambiguous with the comment just `Deprecated` (since there are two names on that line).  Maybe use a longer sentence as explanation?

Reply:
Oops, it should be `MPI_T_ERR_INVALID_INDEX`. Thanks!
Yes I will explain more in a comment.

Reply:
It has been fixed.